src/components/AccountForm/accountForm.js

```diff
@@ -1,38 +1,71 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import BaseForm from '../BaseForm/baseForm';
 import PropTypes from 'prop-types';
+import { useAuth } from '../../utils/Auth/use-auth';
+import { postAccount } from '../../services/Account/account';
+import { updateAccount } from '../../services/Account/account';
 
-const AccountForm = ({ accountId = null }) => {
-    const [formAccount, setFormAccount] = useState({
-        name: '',
-        amount: 0,
-        type: 1
-    });
+
+const emptyAccount = {
+    uid: '',
```

Comment:
Minor suggestion: you could remove this field since we are not using it at this point and has no value